packages/share_plus/share_plus/android/src/main/kotlin/dev/fluttercommunity/plus/share/Share.kt

```diff
@@ -65,21 +65,22 @@ internal class Share(
             }
         }
         // If we dont want the result we use the old 'createChooser'
-        val chooserIntent = if (withResult) {
-            // Build chooserIntent with broadcast to ShareSuccessManager on success
-            Intent.createChooser(
-                shareIntent,
-                null, // dialog title optional
-                PendingIntent.getBroadcast(
-                    context,
-                    0,
-                    Intent(context, SharePlusPendingIntent::class.java),
-                    PendingIntent.FLAG_UPDATE_CURRENT or immutabilityIntentFlags
-                ).intentSender
-            )
-        } else {
-            Intent.createChooser(shareIntent, null /* dialog title optional */)
-        }
+        val chooserIntent =
+            if (withResult && Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP_MR1) {
```

Comment:
Add version check to not see lint errors when working with Android part. 
This change influences nothing, because we have the same check in the method call handler class.